bin/^3.3.x-dev/prepare_project_edition.sh

```diff
@@ -71,6 +71,24 @@ fi
 
 if [[ "$PROJECT_EDITION" != "oss" ]]; then
     composer config repositories.ibexa composer https://updates.ibexa.co
+
+    editions=(commerce experience content)
+
+    IBEXA_PACKAGES="[]"
+    for EDITION in "${editions[@]}"; do
+        if [[ "$PROJECT_EDITION" == "$EDITION" ]]; then
+            break
+        fi
+        BRANCH=${GITHUB_BASE_REF:-"$GITHUB_REF_NAME"} # Fallback to GITHUB_REF for non-PR builds
+        COMPOSER_JSON_CONTENT=$(curl -s "https://raw.githubusercontent.com/ibexa/$EDITION/$BRANCH/composer.json")
+        EDITION_PACKAGES=$(echo "$COMPOSER_JSON_CONTENT" | \
+            jq -r --arg projectEdition "ibexa/$PROJECT_EDITION" \
+            '.require | with_entries(select(.key | contains("ibexa/") or contains("ezsystems/"))) | with_entries(select(.key == $projectEdition | not )) | keys')
```

Comment:
There is one small difference here - we need to take into account packages starting with "ezsystems" (for v4 we only care abour "ibexa"